### PR TITLE
Update scoping doc from phrased as a future proposal to being phrased as statement of current state

### PR DESCRIPTION
**Pull request by dthaler**
Also add other relevant orgs:
TrustedFirmware.org which has been discussed on the TAC mailing list,
and FIDO Alliance which has been discussed in the IETF as being relevant
in this space.

Signed-off-by: Dave Thaler <dthaler@ntdev.microsoft.com>

### CODE REVIEW

File: scoping.md [hunk 1]
```diff
@@ -53,41 +32,59 @@ composed the following Venn diagram of technologies:
...
+The solid blue lines indicate that the area is clearly in scope
+for CCC discussion and projects. The dotted blue lines indicate that
+other areas would not normally fall under the CCC's definition
+of Confidential Computing, and would be in scope of the CCC only
+if a project in the area could be argued to "promote the widespread use of"
+Confidential Computing.  For example, a library that performed
+encryption/decryption inside a hardware-based TEE, for use with
+Homomorphic Encryption, would be relevant to the CCC.
+
+Unfortunately, unlike the term Confidential Computing� several of
```
Comment (MikeCamel) [inline scoping.md:44]: typo after "Computing"

**Review by MikeCamel — CHANGES_REQUESTED** [review]
Good improvements, thanks.

**Comment by dthaler** [comment]
@MikeCamel fixed typo, please check and approve if looks ok.

**Review by spjohnso — CHANGES_REQUESTED** [review]
Some of section 2 on fine grained questions is still written in the past tense:

for instance: the "TAC has the consensus to recommend", or
"All five of these are strategy questions fort he board to answer..."

**Comment by dthaler** [comment]
@spjohnso Updated section 2, please check

**Review by spjohnso — APPROVED** [review]

**Comment by dthaler** [comment]
Time bomb expires October 24 or when a third TAC member approves

**Comment by dthaler** [comment]
Merging per time bomb expiry

**Pull request merged**